PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -316,11 +335,17 @@ void PMTraceConsumer::HandleDxgkQueueSubmit(
         TRACK_PRESENT_PATH(eventIter->second);
         DebugModifyPresent(*eventIter->second);
 
+        // TODO: What happens if there already is an item in mPresentsBySubmitSequence?
+        // We should assert if that's impossible, or remove the existing event from other tracking lists.
         eventIter->second->QueueSubmitSequence = submitSequence;
         mPresentsBySubmitSequence.emplace(submitSequence, eventIter->second);
 
         if (eventIter->second->PresentMode == PresentMode::Hardware_Legacy_Copy_To_Front_Buffer && !supportsDxgkPresentEvent) {
+            // TODO: What happens if there already is an item in mBltsByDxgContext?

```

Comment:
It's impossible. Look at the top of this function, if there was one there, it was already removed.